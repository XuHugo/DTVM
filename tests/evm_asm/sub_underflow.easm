// 0 - 1 = 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
PUSH1 0x01
PUSH0
SUB
PUSH1 0x00
MSTORE
PUSH1 0x20
PUSH1 0x00
RETURN
