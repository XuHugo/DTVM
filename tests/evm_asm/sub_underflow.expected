status: success
error_code: 0
stack: []
memory: FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
storage: {}
transient_storage: {}
return: FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
events: []
